src/common-principles.md

```diff
@@ -536,12 +554,40 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information, such as physiological recordings, MUST be stored with
+[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions) when
+so established by the specifications.
+Rules for formatting plain-text tabular files apply to TSVGZ files with three exceptions:
+
+1.  The contents of TSVGZ files MUST be compressed with
+    [gzip](https://datatracker.ietf.org/doc/html/rfc1952).
+1.  Compressed tabular files MUST NOT contain a header in the first row
+    indicating the column names.
+1.  TSVGZ files MUST be accompanied by a JSON file with the same name as their
+    corresponding tabular file but with a `.json` extension.
```

Comment:
This seems to ignore the inheritance principle. Are you saying that the inheritance principle doesn't apply to `.tsv.gz` files' metadata?

Reply:
It seems like the admonition's text about column names in the JSON file should go here.

Reply:
> This seems to ignore the inheritance principle. Are you saying that the inheritance principle doesn't apply to `.tsv.gz` files' metadata?

Can you suggest an alternative phrasing that makes it clear that these files also follow the inheritance principle?

Reply:
Perhaps

```suggestion
1.  TSVGZ files MUST have an associated JSON file that defines the columns in the tabular file.
```